pkg/p2p/peer/peer.go

```diff
@@ -264,7 +266,9 @@ func (p *Reader) ReadLoop() {
 			return
 		}
 
-		p.router.Collect(bytes.NewBuffer(message))
+		// TODO: error here should be checked in order to decrease reputation
+		// or blacklist spammers
+		_ = p.router.Collect(message)
```

Comment:
Would be good to at least log it already, as I had some issues in `router.Collect` pass me by due to dropping this error